bazel/cloudbuild.yaml

```diff
@@ -1,5 +1,5 @@
 # In this directory, run the following command to build this builder.
-# $ gcloud builds submit . --config=cloudbuild.yaml
+# $ gcloud builds submit
```

Comment:
no need for the source / config?

Reply:
No need -- the default values are no ``config=cloudbuild.yaml` and `.`